tests/phpunit/tests/editor/disableBlockEditorForNavigationPostType.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * @group editor
+ *
+ * @covers ::_disable_block_editor_for_navigation_post_type
+ */
+class Tests_Editor_DisableBlockEditorForNavigationPostType extends WP_UnitTestCase {
+
+	const NAVIGATION_POST_TYPE     = 'wp_navigation';
+	const NON_NAVIGATION_POST_TYPE = 'wp_non_navigation';
+
+	/**
+	 * @ticket 56266
+	 *
+	 * @dataProvider data_test_should_only_disable_block_editor_for_navigation_post_types
+	 *
+	 * @param string $post_type The post type.
+	 * @param bool   $value     Whether to disable the block editor.
+	 * @param bool   $expected  The expected result.
+	 */
+	public function test_should_only_disable_block_editor_for_navigation_post_types( $post_type, $value, $expected ) {
+		$filtered_result = _disable_block_editor_for_navigation_post_type( $value, $post_type );
+		$this->assertSame( $expected, $filtered_result );
+	}
+
+	/**
+	 * Data provider for test_should_only_disable_block_editor_for_navigation_post_types().
+	 *
+	 * @return array
+	 */
+	public function data_test_should_only_disable_block_editor_for_navigation_post_types() {
```

Comment:
```suggestion
	public function data_should_only_disable_block_editor_for_navigation_post_types() {
```

Reply:
Fixed in d6ee08b6441d289e4009cbf66b298f3959fa54ad